.github/workflows/ci.yml

```diff
@@ -241,13 +218,15 @@ jobs:
         timeout-minutes: 5
         env:
           TEST_NAME: ${{ matrix.test_name }}
-        run: ./scripts/tests/perf/${{ matrix.test_name }}.sh
+        run: |
+          ./scripts/tests/perf/${{ matrix.test_name }}.sh
+          jq '{ "${{ matrix.test_name }}": { "throughput": { "value": (.end.sum_received.bits_per_second / 8 / 1000 / 1000) } } }' ./${{ matrix.test_name }}.json > ./${{ matrix.test_name }}.bmf.json
       - name: 'Save performance test results: ${{ matrix.test_name }}'
         uses: actions/upload-artifact@v4
         with:
           overwrite: true
-          name: ${{ matrix.test_name }}-${{ matrix.version.sha }}-iperf3results
-          path: ./${{ matrix.test_name }}.json
+          name: ${{ matrix.test_name }}-${{ github.sha }}-iperf3results
+          path: ./${{ matrix.test_name }}.bmf.json
```

Comment:
`bmf` is some JSON schema specific to bencher.dev?

Reply:
Yes.

https://bencher.dev/docs/reference/bencher-metric-format/